canarytokens/windows_fake_fs.py

```diff
@@ -0,0 +1,179 @@
+from datetime import datetime
+import random
+from math import floor
+from io import StringIO
+from csv import DictWriter
+
+from canarytokens.models import Hostname
+from canarytokens.windows_fake_fs_templates.powershell_template import (
+    POWERSHELL_TEMPLATE,
+)
+from canarytokens.windows_fake_fs_templates.folder_personal_finances import (
+    PERSONAL_FINANCES_STRUCTURE,
+)
+from canarytokens.windows_fake_fs_templates.folder_home_network import (
+    HOME_NETWORK_STRUCTURE,
+)
+from canarytokens.windows_fake_fs_templates.folder_personal_correspondence import (
+    PERSONAL_CORRESPONDENCE_STRUCTURE,
+)
+from canarytokens.windows_fake_fs_templates.folder_photo_archive import (
+    PHOTO_ARCHIVE_STRUCTURE,
+)
+from canarytokens.windows_fake_fs_templates.folder_defense import (
+    DEFENSE_STRUCTURE,
+)
+from canarytokens.windows_fake_fs_templates.folder_med_tech import MED_TECH_STRUCTURE
+from canarytokens.windows_fake_fs_templates.folder_network_admin import (
+    NETWORK_ADMIN_STRUCTURE,
+)
+from canarytokens.windows_fake_fs_templates.folder_security_admin import (
+    SECURITY_ADMIN_STRUCTURE,
+)
+
+
+DUMMY_FOLDER_STRUCTURE = [
```

Comment:
Is there a reason for this to be in this file instead of its own file like the other folder structures?